### ISSUE TITLE
css example?

### ISSUE THREAD

**Issue by deadcyclo** [rotty3000/blade#260]
There probably should be an example of using css in a portlet here

**Comment by deadcyclo** [comment]
I actually got this working, but I have a feeling I might not be doing it correctly. I'll submit a push request, and see what you guys think.

**Comment by gamerson** [comment]
Yes we do need an example of CSS in a portlet.  send us what you have and
we can review.

On Wed, Mar 16, 2016 at 8:16 AM, Brendan Johan Lee <notifications@github.com
> wrote:

> I actually got this working, but I have a feeling I might not be doing it
> correctly. I'll submit a push request, and see what you guys think.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/rotty3000/blade/issues/260#issuecomment-197323137>
>



-- 
Greg Amerson
Liferay Developer Tools
Liferay, Inc. www.liferay.com

**Comment by deadcyclo** [comment]
I sort of got it working, but only in my own freemarker portlet. When I moved it to a jsp portlet it stopped working, so need to do some more investigations.
